### PR TITLE
Fixed #25894 -- Fixed evaluation of zero-length slices of queryset.values().

### PR DESCRIPTION
**Pull request by sir-sigurd**
https://code.djangoproject.com/ticket/25894

### CODE REVIEW

File: docs/releases/1.9.1.txt [hunk 1]
```diff
@@ -19,3 +19,5 @@ Bugfixes
 
 * Fixed a state bug when migrating a ``SeparateDatabaseAndState`` operation
   backwards (:ticket:`25896`).
+
+* Fixed evaluation of zero-length slices of queryset.values() (:ticket:`25894`).
```
Comment (timgraham) [inline docs/releases/1.9.1.txt]: QuerySet.values() and add double backticks around it

**Comment by timgraham** [comment]
@akaariai, could you double check? (code removed here was added in afe0bb7b)

**Comment by akaariai** [comment]
LGTM

**Comment by timgraham** [comment]
merged in 69b69f6d6085d80ccf36a3e999e9e5b98d580786, thanks!

**Pull request closed without merging**